migration_original/ODS1Stage/tables/Base/ProviderToDegree/spu_translated_ProviderToDegree.sql

```diff
@@ -0,0 +1,98 @@
+CREATE OR REPLACE PROCEDURE ODS1_STAGE.BASE.SP_LOAD_PROVIDERTODEGREE()
+RETURNS STRING
+LANGUAGE SQL EXECUTE
+AS CALLER
+AS DECLARE 
+
+---------------------------------------------------------
+--------------- 0. Table dependencies -------------------
+---------------------------------------------------------
+-- Base.ProviderToDegree depends on:
+-- Raw.Provider_Profile_JSON
```

Comment:
Replace this with the new name Raw.VW_PROVIDER_PROFILE
```suggestion
-- Raw.VW_PROVIDER_PROFILE
```

Reply:
Done and used degree abbreviation for the join